Covid19Radar/Covid19Radar.iOS/Services/ExposureNotificationApiService.cs

```diff
@@ -0,0 +1,86 @@
+﻿/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Chino;
+using Chino.iOS;
+using Covid19Radar.Services;
+using Covid19Radar.Services.Logs;
+
+namespace Covid19Radar.iOS.Services
+{
+    public class ExposureNotificationApiService : AbsExposureNotificationApiService
+    {
+        private ExposureNotificationClient _exposureNotificationClient = new ExposureNotificationClient();
+
+        public string UserExplanation
+        {
+            set
+            {
+                _exposureNotificationClient.UserExplanation = value;
+            }
+        }
+
+        public bool IsTest
+        {
+            set
+            {
+                _exposureNotificationClient.IsTest = value;
+            }
```

Comment:
[修正任意]
アロー演算子を使うとスッキリ書けます。

```suggestion
            set => _exposureNotificationClient.IsTest = value;
```

Reply:
6a3cbf0 で対応しました！